connector/avro/src/main/scala/org/apache/spark/sql/v2/avro/AvroPartitionReaderFactory.scala

```diff
@@ -25,9 +25,9 @@ import org.apache.avro.mapred.FsInput
 import org.apache.spark.TaskContext
 import org.apache.spark.broadcast.Broadcast
 import org.apache.spark.internal.Logging
-import org.apache.spark.sql.avro.{AvroDeserializer, AvroOptions, AvroUtils}
 import org.apache.spark.sql.catalyst.{InternalRow, NoopFilters, OrderedFilters}
 import org.apache.spark.sql.connector.read.PartitionReader
+import org.apache.spark.sql.core.avro.{AvroDeserializer, AvroOptions, AvroUtils}
```

Comment:
Do we need to change the package name or we could keep the same name and just move the files ?